WarshipGame/Assets/Scripts/Gameplay/Grid/HexData.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using UnityEngine;
+
+public enum HexType
+{
+    Default,
```

Comment:
`Default` can probably be made more clear by naming it `Free`

Reply:
nah